2021-10-18 14:37:28,112 INFO    MainThread:129 [wandb_setup.py:_flush():69] setting env: {}
2021-10-18 14:37:28,113 INFO    MainThread:129 [wandb_setup.py:_flush():69] setting login settings: {'api_key': 'eb6f1b610f9f4bb9f1bd7c8e03173a84ea25a050'}
2021-10-18 14:37:28,113 INFO    MainThread:129 [wandb_setup.py:_flush():69] setting login settings: {'api_key': 'eb6f1b610f9f4bb9f1bd7c8e03173a84ea25a050'}
2021-10-18 14:37:28,113 INFO    MainThread:129 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-18 14:37:28,113 INFO    MainThread:129 [wandb_init.py:_log_setup():338] Logging user logs to /home/ml2558/CacheSimulator/src/wandb/run-20211018_143728-21kjrouh/logs/debug.log
2021-10-18 14:37:28,113 INFO    MainThread:129 [wandb_init.py:_log_setup():339] Logging internal logs to /home/ml2558/CacheSimulator/src/wandb/run-20211018_143728-21kjrouh/logs/debug-internal.log
2021-10-18 14:37:28,113 INFO    MainThread:129 [wandb_init.py:init():371] calling init triggers
2021-10-18 14:37:28,114 INFO    MainThread:129 [wandb_init.py:init():378] wandb.init called with sweep_config: {}
config: {'test_randomize_ordering_seed': '', 'pdb_post_mortem': False, 'only_check_args': False, 'alsologtostderr': False, 'profile_file': None, 'use_cprofile_for_profiling': True, 'logtostderr': False, 'showprefixforinfo': True, 'log_dir': '', 'test_srcdir': '', 'run_with_profiling': False, 'test_tmpdir': '/tmp/absl_testing', 'op_conversion_fallback_to_while_loop': False, 'stderrthreshold': 'fatal', 'test_random_seed': 301, 'run_with_pdb': False, 'verbosity': -1, 'v': -1, 'xml_output_file': ''}
2021-10-18 14:37:28,114 INFO    MainThread:129 [wandb_init.py:init():420] starting backend
2021-10-18 14:37:28,114 INFO    MainThread:129 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-18 14:37:28,126 INFO    MainThread:129 [backend.py:ensure_launched():135] starting backend process...
2021-10-18 14:37:28,132 INFO    MainThread:129 [backend.py:ensure_launched():140] started backend process with pid: 148
2021-10-18 14:37:28,133 INFO    MainThread:129 [wandb_init.py:init():425] backend started and connected
2021-10-18 14:37:28,139 INFO    MainThread:129 [wandb_init.py:init():473] updated telemetry
2021-10-18 14:37:28,139 INFO    MainThread:129 [wandb_init.py:init():492] communicating current version
2021-10-18 14:37:28,824 INFO    MainThread:129 [wandb_init.py:init():497] got version response upgrade_message: "wandb version 0.12.4 available, but requires python3.6+, see http://wandb.me/library-eol-python35"

2021-10-18 14:37:28,825 INFO    MainThread:129 [wandb_init.py:init():505] communicating run to backend with 30 second timeout
2021-10-18 14:37:29,012 INFO    MainThread:129 [wandb_init.py:init():530] starting run threads in backend
2021-10-18 14:37:30,762 INFO    MainThread:129 [wandb_run.py:_console_start():1625] atexit reg
2021-10-18 14:37:30,763 INFO    MainThread:129 [wandb_run.py:_redirect():1499] redirect: SettingsConsole.REDIRECT
2021-10-18 14:37:30,763 INFO    MainThread:129 [wandb_run.py:_redirect():1504] Redirecting console.
2021-10-18 14:37:30,766 INFO    MainThread:129 [wandb_run.py:_redirect():1560] Redirects installed.
2021-10-18 14:37:30,766 INFO    MainThread:129 [wandb_init.py:init():555] run started, returning control to user process
2021-10-18 14:39:27,612 INFO    Dummy-12  :129 [wandb_run.py:_atexit_cleanup():1595] got exitcode: 255
2021-10-18 14:39:27,612 INFO    Dummy-12  :129 [wandb_run.py:_restore():1567] restore
